packages/fx-core/src/plugins/resource/aad/aadAppClient.ts

```diff
@@ -218,6 +229,7 @@ export class AadAppClient {
         if (retries === 0) {
           throw error;
```

Comment:
When `throw error`, need also add telemetry?

Reply:
Here will cause fail, and telemetry will be sent in index.ts

Reply:
The telemetry sent in index.ts would not have info of retry times. I think we'd better to add the info as well? Otherwise, I may miss the telemetry of retry times is 10.

Reply:
Oh, just see that we will send telemetry every time it is retrying. So we may not need to do that.